src/ui/component/DisciteFeedback.html

```diff
@@ -0,0 +1,42 @@
+<div class="discite__item u-clearfix ">
+    <div class="discite__main discite__main--module ">
+        <a href="/courses/{{course.id}}/{{module.id}}">
+            <h3 class="heading bold-small heading--text">
+                {{#if module.title}}
+                {{module.title}}
+                {{else}}
+                {{course.title}}
+                {{/if}}
+            </h3>
+        </a>
+        {{#if module.description}}
+        <p class="discite__description">{{module.description}}</p>
+        {{else}}
+        <p class="font-small discite__counter">
+            This is a module in <a href="/courses/{{course.id}}"><b>{{ course.title }}.</b></a>
```

Comment:
This isn't always the case - if it's a single module course then I assume you should show the course shortDescription.